mobly/controllers/android_device_lib/services/snippet_management_service.py

```diff
@@ -69,7 +69,7 @@ def add_snippet_client(self, name, package, config=None):
         class for supported configurations.
 
     Raises:
-      Error, if a duplicated name or package is passed in.
+      Error: if a duplicated name or package is passed in.
```

Comment:
the error message here should be updated as package name alone is no longer a differentiating factor?